h2o-automl/src/main/java/ai/h2o/automl/modeling/XGBoostStepsProvider.java

```diff
@@ -193,7 +18,19 @@ public String getName() {
 
     @Override
     public XGBoostSteps newInstance(AutoML aml) {
-        return new XGBoostSteps(aml);
+        return Algo.XGBoost.enabled() ? new XGBoostSteps(aml) : null;
+    }
+
+    @Override
+    public Model.Parameters newDefaultParameters() {
+        if (Algo.XGBoost.enabled()) {
+            try {
```

Comment:
There is a generic way of constructing new ModelBuilder instance and parameters, you could do 

```
   ModelBuilder.make("xgboost", null, null)._parms;
```

Reply:
Is it worth it? Creating a builder instance just to get the params?

Reply:
checked the impl: yes, it's probably better, will switch. Thx